src/lib/generators/server/generate-server.ts

```diff
@@ -402,13 +403,13 @@ export const generateServer = (ui: UI): OptionalKind<SourceFileStructure> => {
             statements: [
               "if (process.env.UPLOAD_PROVIDER === 'local') {\r\n  this.express.use('/uploads', express.static(process.env.UPLOAD_PROVIDER_FOLDER));\r\n}",
               "this.express.use('/', new RootRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/auth`, new AuthRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/users`, new UserRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/resource-permissions`, new ResourcePermissionRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/roles`, new RoleRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/media-objects`, new MediaObjectRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/endpoints`, new EndpointRoute().router);",
-              "this.express.use(`/${process.env.API_NAME}/books`, new BookRoute().router);",
+              "this.express.use(`/${this.api}/auth`, new AuthRoute().router);",
+              "this.express.use(`/${this.api}/users`, new UserRoute().router);",
+              "this.express.use(`/${this.api}/resource-permissions`, new ResourcePermissionRoute().router);",
+              "this.express.use(`/${this.api}/roles`, new RoleRoute().router);",
+              "this.express.use(`/${this.api}/media-objects`, new MediaObjectRoute().router);",
+              "this.express.use(`/${this.api}/endpoints`, new EndpointRoute().router);",
+              "this.express.use(`/${this.api}/books`, new BookRoute().router);",
```

Comment:
This is related to .env
You can take a look in .env.sample for more info.

Reply:
`this.api` is equivalent to `${process.env.API_NAME}` and already defined in the constructor.